rfc-139-enable-http3.md

```diff
@@ -0,0 +1,135 @@
+
+The data shows that there are parts of the UK that have connection speeds less than 0.3 MBit/s for both download, and upload. This is roughly equivalent to a 2G connection. It's worth noting that as these are median values, many users in these areas will be seeing much lower connection speeds than this in reality. These areas have a connection speed that is 98.5% slower than the median for the whole country. 
+
+Users in these areas still have a need to access government services and the information that GOV.UK offers, but under these conditions it won't be a very pleasant experience. These are the users that will benefit from a transition to HTTP/3.
+
+## Brief history of HTTP
+Each iteration of the HTTP protocol has set out to improve the web performance of the previous version of the protocol.
+
+### HTTP/1.0
+One of the first versions of the HTTP protocol came with a number of web performance issues. A primary issue being that only a single asset could be download on a Transmission Control Protocol (TCP) connection before the connection was then closed. There was no way for these connections to persist for multiple asset downloads. This resulted in a lot of waisted negotiation time to establish separate TCP connections for each asset.
+
+### HTTP/1.1
+HTTP/1.1 fixed the persistent connections issue, allowing multiple assets to be downloaded over a single connection once established. Unfortunately this didn't fully solve the web performance issues seen with the HTTP protocol (stemming from the TCP layer). 
+
+Under HTTP/1.1 the browser must download 100% of an asset before it can move onto the next asset. Meaning it can only download a single asset at a time per connection, the other assets are blocked. This is called Head-of-Line Blocking (HOL Blocking). Browser vendors quickly tried to solve this issue by allowing a browser to open up to 6 TCP connections per domain, so 6 assets can be downloaded in parallel.
+
+Unfortunately a number of issues still exist with this method:
+
+1. Most sites have many more than 6 assets per page, so HOL blocking is still occurring in almost every page load under HTTP/1.1.
+2. Each TCP connection is independent and have no knowledge of each other. All 6 are fighting for limited bandwidth which in turn can saturate the network.
+3. Many websites resorted to domain 'sharding', so as to allow a browser to open up more that 6 TCP connections, in turn exacerbating the problem mentioned in point 2.
```

Comment:
May be worth linking to a definition of domain sharding, or inlining a definition here:

```suggestion
3. Many websites resorted to "domain sharding" (hosting assets across multiple domains), to allow a browser to open up more that 6 TCP connections. This in turn exacerbated the problem mentioned in point 2.
```